### ISSUE TITLE
Internal-link autocompletion window hidden

### ISSUE THREAD

**Issue by ssoriche** [adithya-badidey/TW5-codemirror-plus#24]
Thank you for creating this plugin, it's extremely useful.

I'm having an issue with the autocompletion window appearing behind the editor, so I am unable to see the list of tiddler titles to select the one I want. If the cursor close to the right edge of the edit box, I can see part of the list, but otherwise it is completely hidden.

TiddlyWiki version 5.1.23
CMPlus version 0.2.2

![image](https://user-images.githubusercontent.com/19838/130633522-c7559e36-596b-433b-b3b9-8bd89c5f2dd0.png)

**Comment by adithya-badidey** [comment]
Thanks for reporting this. This is probably due to the z-index. I’ll fix this over the weekend.

**Comment by trumad** [comment]
I had to edit $:/plugins/tiddlywiki/codemirror/addon/hint/show-hint.css and change the z-index to 300. The autocomplete was appearing behind tiddlers, and I just couldn't fix it. By using https://addons.mozilla.org/en-US/firefox/addon/devtools-z-index/ I was able to see that tiddlers use auto-generated z-indexes of 194, 195 etc with the tc-tiddler-frame class. I don't know why, but the newest version of TW doesn't seem to use this classname or z-index solution.

**Comment by adithya-badidey** [comment]
Hi!
I tried to replicate the problem but couldn't. Is the problem fixed in the CodeMirror-Enhanced plugin? If yes, great!

**Comment by Gk0Wk** [comment]
Adding this code to `$__plugins_tiddlywiki_codemirror_mode_tw5_tw5.css.tid` can help:

```css
.CodeMirror-hints {
    z-index: 800 !important;
}
```

**Comment by trumad** [comment]
Yeah, it seems to be an issue with old tiddlywikis that were upgraded, or something. It's difficult to pin down. But that bit of CSS will definitely fix it.
